docs/libcurl/opts/CURLOPT_FTP_SKIP_PASV_IP.3

```diff
@@ -45,7 +52,7 @@ abuse by malicious servers.
 
 This option has no effect if PORT, EPRT or EPSV is used instead of PASV.
 .SH DEFAULT
-1 since 7.74.0, was 0 before then.
+CURL_FTP_SKIP_PASV_IP_ALWAYS
```

Comment:
This removes the information about the historic change and I think it should not do that.

Reply:
updated